src/rsa/convert_nist_rsa_test_vectors.py

```diff
@@ -102,7 +102,7 @@ def print_sign_test(case, n, e, d, padding_alg):
         # PSS is randomised, can't recompute this way
         pass
     else:
-        print "Invalid padding algorithm"
```

Comment:
I've added this in a separate commit to make the script compatible with Python 3 as I can't install Python 2.7 from homebrew anymore, but I can remove this if you don't want this.